src/main/java/com/commercetools/sync/products/service/impl/ProductReferenceTransformServiceImpl.java

```diff
@@ -0,0 +1,496 @@
+package com.commercetools.sync.products.service.impl;
```

Comment:
What do you think about moving the services to the "com.commercetools.sync.services.transform" package?

Reply:
Since the class name already contains "Transform" and also we don't have any other service classes as of now. I think it is good to have in com.commercetools.sync.service package itself. Let me know if you think still it is better to have transform package!